app/reducers/settings-default.ts

```diff
@@ -152,7 +152,10 @@ export default {
   useTrashCan: true,
   useOCR: false,
   useTextExtraction: false,
-  useGenerateThumbnails: true,
+  useGenerateThumbnails:
```

Comment:
such code is typically in config.js, see https://github.com/tagspaces/tagspaces/blob/develop/app/config.ts#L68